src/main/java/ro/societateahermes/backendservice/service/ParticipationServiceInterface.java

```diff
@@ -0,0 +1,14 @@
+package ro.societateahermes.backendservice.service;
+
+import ro.societateahermes.backendservice.entities.DTO.MySubmissionDTO;
+import ro.societateahermes.backendservice.entities.Participation;
+import ro.societateahermes.backendservice.entities.User;
+
+import java.util.List;
+
+public interface ParticipationServiceInterface {
+
+    void savefromDTO(User user, MySubmissionDTO submissionDTO);
```

Comment:
De unde vine userul aici? Daca e din frontend trebuie sa fie UserDto

Reply:
Din frontend vine submissionDTO care are campurile necesare si pentru user, salvez user ul( si returnez ceea ce salvez in db pentru a avea si id ul lui, id ul fiind autogenerated) si il transmit mai departe in participationService pentru a salva si participation......in user controlller am codul asta:
 User user = userService.saveUserFromDTO(submission);
 participationService.savefromDTO(user, submission);